s3clientconfig.go

```diff
@@ -2,6 +2,7 @@ package analytics
 
 import (
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws/session"
```

Comment:
Let's organise imports separating by a blank line system packages from 3rd-party packages from internal packages.

So this would become:

```
import (
 	"fmt"
 	"strings"
 	"time"

 	"github.com/aws/aws-sdk-go/aws/session"
 )
```